pkgs/development/libraries/webrtc-audio-processing/default.nix

```diff
@@ -8,6 +8,8 @@ stdenv.mkDerivation rec {
     sha256 = "1gsx7k77blfy171b6g3m0k0s0072v6jcawhmx1kjs9w5zlwdkzd0";
   };
 
+  patches = lib.optional stdenv.hostPlatform.isRiscV ./enable-riscv.patch;
```

Comment:
```suggestion
  patches = [ ./enable-riscv.patch ];
```

Reply:
Sure, let me apply it unconditionally. We cannot fetchpatch as we are shipping an ancient version of the package that cannot be updated yet (#112293). The proposed upstream patch does not apply as the file has been moved.